content/php-client/resources/product-models.md

```diff
@@ -27,3 +27,36 @@ $client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAut
  */
 $productModel = $client->getProductModelApi()->get('rain_boots_red');
 ```
+
+### Create a product model
+
+If the product model does not exist yet, this method creates it, otherwise it throws an exception.
+
+```php
+$client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$client->getProductModelApi()->create([
+  'code' => 'saddle_rain_boots',
+  'family_variant' => 'boots_color_size',
+  'parent' => 'rain_boots',
+  'categories' => ['2014_collection', 'winter_boots', 'winter_collection'],
+  'values' => [
+      'name' => [
+          [
+              'locale' => 'en_US',
+              'scope' => null,
+              'data' => 'Red rain boots',
+          ]
+      ],
+      'color' => [
+          [
+              'locale' => null,
+              'scope' => null,
+              'data' => 'saddle'
+          ]
+      ]
+  ]
+]);
+```
+
+You can get more information about the expected format of the product values [here](/documentation/resources.html#product).
```

Comment:
"product model values" instead of "product values"

Reply:
I would even be more precise by saying:
"Product model values use the same format as the product values. If you want to know more, take a look at [here](/documentation/resources.html#product-values)"
Beware, the link given was wrong too.